plugins/modules/purefa_vnc.py

```diff
@@ -157,7 +161,10 @@ def main():
         module.fail_json(msg="py-pure-client sdk is required for this module")
 
     array = get_array(module)
-
+    api_version = array.get_rest_version()
+    if LooseVersion(MAX_API_VERSION) <= LooseVersion(api_version):
+        module.warn(msg="VNC feature deprecated from Purity//FA 6.8.0 and higher")
```

Comment:
module.warn doesn't need 'msg' variable, should drop it and pass the string directly.